origin-growth/migrations/20190312002247-update-growth-referral.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const tableName = 'growth_referral'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable(tableName)
```

Comment:
It's unusual to drop a table as part of a migration since all legacy data gets lost... but I guess in our case we don't care since there is no data in prod. So that's probably ok :)

Reply:
yup, still a good thing to keep in mind